src/components/Main/Main.js

```diff
@@ -0,0 +1,13 @@
+import { usePosts } from '../../hooks/usePosts';
+import BlogCard from '../BlogCard/BlogCard';
+
+export default function Main() {
+  const posts = usePosts();
+  return (
+    <>
+      {posts.map(({ id, title, subtitle, text, image }) => (
+        <BlogCard key={id} {...{ title, subtitle, text, image }} />
```

Comment:
What you're doing here works, but there is no need for the destructuring of the object in the .map. It's more than you need to write and harder to read. Consider changing these lines to the following:

```js
{posts.map((post) => (
        <BlogCard key={id} {...post} />